x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/model_select_logic.ts

```diff
@@ -167,16 +146,16 @@ export const ModelSelectLogic = kea<MakeLogicType<ModelSelectValues, ModelSelect
       },
     ],
     selectableModels: [
-      () => [selectors.modelsData],
-      (response: FetchModelsApiResponse) => response ?? [],
+      () => [selectors.selectableModelsFromMLInferenceLogic],
+      (selectableModels) => selectableModels, // Pass-through
```

Comment:
I wonder if we need these pass-through selectors, or we should just expose the imported selectors directly like [startPollingModels](https://github.com/elastic/kibana/pull/176257/files#diff-83c7e40124c2eeb3cfbd4081cb93dc8cca1afab4f7ffa53e4d2136e35b8edf10R81).

Reply:
Discussed with @TattdCodeMonkey that we don't need plain pass-through selectors. I'll remove these in a follow-up PR.